app/api/route.tsx

```diff
@@ -1,34 +1,28 @@
 import { fetchUsers } from "@/database/fetch";
 import { NextRequest, NextResponse } from "next/server";
-
-export async function GET(request: NextRequest, response: NextResponse) {
-	try {
-		const users = await fetchUsers();
-		return Response.json(users);
-	} catch (error) {
-		console.error("Error in GET users request:", error);
-	}
-}
+import { serialize } from "cookie";
 
 export async function POST(request: NextRequest, response: NextResponse) {
 	try {
 		const users = await fetchUsers();
 		const data = await request.json();
 		const { email, password } = data;
-		const doesUserExists = users.find(
-			(user) => user.email === email && user.password === password
-		);
-		const existingEmail = users.find((user) => user.email === email);
-		const existingPassword = users.find((user) => user.password === password);
+		const user = users.find((user) => user.email === email);
 
-		if (existingEmail && existingPassword) {
-			return Response.json("success");
-		} else if (existingEmail && !existingPassword) {
-			return Response.json("Invalid password");
-		} else if (!existingEmail && existingPassword) {
+		if (user === undefined) {
```

Comment:
metoda find, jeśli nie znajdzie wartości zwraca undefined, nie ma potrzeby przyrównywać tego, można zapisać prościej

```suggestion
		if (!user) {
```